ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -184,7 +220,9 @@ objects:
           exit 0
         }
 
-        ${TEST_COMMAND}
+        #FIXME: hard wire this to run parallel tests
```

Comment:
NOTE: this will get removed once I see if this works. I wanted to hard wire it here as we don't run full tests in most places yet (just the canary jobs)